### ISSUE TITLE
Week 27: Learn GoLang For Great Good Part 1

### ISSUE THREAD

**Issue by shekhargulati** [shekhargulati/52-technologies-in-2016#36]
Please provide your feedback by posting a comment against this issue.

**Comment by thewhitetulip** [comment]
I have written an example based book to teach intro to writing web apps in Go, https://github.com/thewhitetulip/web-dev-golang-anti-textbook/

**Comment by shekhargulati** [comment]
@thewhitetulip thanks for sharing the link. This month I am working on Go programming language. Would you like to contribute a Go web programming post to this project?

**Comment by thewhitetulip** [comment]
Sure, if you want you can copy paste the content of the book directly and link it to the book

**Comment by shekhargulati** [comment]
@thewhitetulip thanks for copy paste offer :) . Actually, I will not like to do that as then your book will have two source of truth. If you have time, can you create a small post based out of your book? We will add a link to your book on the blog. This way people will go to your book to read in detail.

**Comment by thewhitetulip** [comment]
Yes, post won't be a problem. How do you want the post to be? how long, what contents?

**Comment by shekhargulati** [comment]
@thewhitetulip I will write a doc explaining how people can contribute and send you the link.

**Comment by thewhitetulip** [comment]
Okay, meanwhile I'll write a small post about how to write webapps too

**Comment by exu** [comment]
Hi, I'm missing a lot in Learn Golang, Go has some important advantages like tooling (go command), really powerfull testing (with great benchmarking), concurrency is really easy great with use of channels. Object model is really great with structs, you can do nice things with it. Whole network programming is quite easy with Go. Marshaling and unmarshaling data.  and really many many more. Feel free to look (and use if you want) at programs written for My Go training: https://github.com/exu/go-workshops

**Comment by shekhargulati** [comment]
@exu I have only written part 1 of Golang. I am working on second post and possibly write 5 posts on Golang. There will be 2 posts contributed by community as well. So, I am expecting in a month or two we will have good coverage. Do you like to contribute a post to this series?
